runtime/dolphin/src/weights/pallet_assets.rs

```diff
@@ -74,13 +75,13 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_assets::WeightInfo for SubstrateWeight<T> {
     // Storage: Assets Asset (r:1 w:1)
     fn create() -> Weight {
-        (20_622_000 as Weight)
+        (13_228_000 as Weight)
```

Comment:
creating an asset became 40% cheaper?

Reply:
For some reason, the v3.2.0 release increased this to 20M.
It was ~15M before.

Dunno why chaos exists @Dengjianping since this weight updating process is well organized, but at least the new value is not an outlier.
I can't remember us ever talking about this in v3.2.0, so the manual detection of benchmarking changes is not reliable it seems.

Reply:
From latest result:
```rust
fn create() -> Weight {
        (13_450_000 as Weight)
            .saturating_add(T::DbWeight::get().reads(1 as Weight))
            .saturating_add(T::DbWeight::get().writes(1 as Weight))
    }
```

Reply:
We haven't benchmarked until now so now is the time to discuss these.
https://github.com/Manta-Network/Manta/issues/399